includes/Mesh.h

```diff
@@ -174,13 +178,13 @@ class Mesh {
     std::vector<Triangle *> triangles;
     std::vector<Material> materials;
     std::vector<Chunk> m_chunks;
-    aiMatrix4x4 globalInverseTransform;
 
+    BoneTransformer *boneTransformer;
```

Comment:
Is it possible to use a smart pointer such as `std::shared_ptr<BoneTransformer>` or `std::unique_ptr<BoneTransformer>` instead ?